next/components/forms/widget-wrappers/RadioButtonWidgetRJSF.tsx

```diff
@@ -48,12 +48,19 @@ const RadioButtonsWidgetRJSF = (props: RadioButtonFieldWidgetRJSFProps) => {
   const getTooltip = (radioValue: string) => {
     return radioOptions.find((option) => option.value === radioValue)?.tooltip
   }
+
+  const handleOnChange = (data: string) => {
+    const option = enumOptions.find((enumOption: EnumOptionsType) => enumOption.value === data)
+    const newValue: string|boolean|number = option ? option.value : data
```

Comment:
it's not clear why setting the newValue to `data` that is not part of the available enum options should be a thing - if it is indeed needed, please add a comment detailing why it should behave this way and in what situation this is useful

Reply:
i forgot it. originally i was thinking it is needed